projectEuler4_final.py

```diff
@@ -0,0 +1,36 @@
+#ProjectEuler Question 4
+#A palindromic number reads the same both ways. The largest palindrome made from the product of two 2-digit numbers is 9009 = 91 * 99.
+#Find the largest palindrome made from the product of two 3-digit numbers.
+
+def isPalindrome(number):
+    strNum = str(number)
+    for i in range(0, len(strNum)/2):
+        if strNum[i] != strNum[len(strNum) - 1 - i]:
+            return False
+    return True
+
+def largestProduct(num1, num2):
+    largest = 0
+    while num1 > 99 and num2 > 99:
+        x = 0
+        product = num1 * num2        
+        while num1 > 99 and num2 > 99:
```

Comment:
this is weird since you only decrements num1 in this loop, meaning num2 will never be less than 99